source/Halibut.Tests/BackwardsCompatibility/Util/ClientAndOldServiceBuilder.cs

```
@@ -0,0 +1,101 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Halibut.Tests.Util;
+
+namespace Halibut.Tests.BackwardsCompatibility.Util
+{
+    public class ClientAndOldServiceBuilder
+    {
+        readonly ServiceConnectionType serviceConnectionType;
+        readonly CertAndThumbprint serviceCertAndThumbprint;
+        readonly CertAndThumbprint clientCertAndThumbprint = CertAndThumbprint.Octopus;
+        string version = "5.0.429";
```

Throw instead of default